modules/sandboxed-containers-common-terms.adoc

```diff
@@ -0,0 +1,34 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+[id="sandboxed-containers-common-terms_{context}"]
+
+= Common terms
+
+.Sandbox
```

Comment:
Thought: this would be a great place for [description lists](https://asciidoctor.org/docs/asciidoc-writers-guide/#description-lists).

Reply:
This is a great idea! I didn't think about description list. I think this is a good idea as well.